drv/lpc55-spi-server/src/main.rs

```diff
@@ -4,27 +4,10 @@
 
 //! A driver for the LPC55 HighSpeed SPI interface.
 //!
-//! Mostly for demonstration purposes, write is verified read is not
```

Comment:
Not exactly related to this PR, but I find myself confused as to what this driver is doing!  Is it actually a SPI server, or still in a development state where it's just sending the same buffers over and over?  A comment to that effect would be helpful!

Reply:
This is still in a development state. Once we get the RoT -> SP communication finalized it will certainly be more useful.